scipy/spatial/distance.py

```diff
@@ -1050,7 +1049,7 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
        Computes the Jaccard distance between the points. Given two
        vectors, ``u`` and ``v``, the Jaccard distance is the
        proportion of those elements ``u[i]`` and ``v[i]`` that
-       disagree where at least one of them is non-zero.
+       disagree.
```

Comment:
I think that this was hinting at the fact that two NaNs are never equal to each other.